gradle/artifact-settings.gradle

```
@@ -10,5 +10,5 @@ ext {
     mapboxArtifactScmUrl = 'scm:git@github.com:mapbox/mapbox-navigation-android.git'
     mapboxArtifactLicenseName = 'Mapbox Terms of Service'
     mapboxArtifactLicenseUrl = 'https://www.mapbox.com/legal/tos/'
-    versionName = System.getenv('POM_VERSION_NAME') ?: project.property('POM_SNAPSHOT_VERSION_NAME')
+    versionName = project.hasProperty('VERSION_NAME') ? project.property('VERSION_NAME') : project.property('SNAPSHOT_VERSION_NAME')
```

```suggestion
    versionName = project.hasProperty('VERSION_NAME') ? project.property('VERSION_NAME') : System.getenv('VERSION_NAME')
```
To resolve the issue with `sed`.